packages/dotcom-ui-shell/src/components/Shell.tsx

```diff
@@ -32,8 +32,8 @@ type TShellProps = TDocumentHeadProps &
 
 function Shell(props: TShellProps) {
   const bootstrapProps: TBootstrapProps = {
-    coreScripts: [polyfillService.core()],
-    enhancedScripts: [polyfillService.enhanced(), ...props.scripts]
+    coreScripts: [],
```

Comment:
**Question:** You planning to do the clean up of `@financial-times/dotcom-ui-polyfill-service` afterwards?

Reply:
Does [line 15](https://github.com/Financial-Times/dotcom-page-kit/pull/1026/files#diff-07a5a1f5a61d8361b19ebcc04b34f55c76561580f67451783776e7d7e4523d8cR15) need removing to avoid unused warnings?

Reply:
yea

Reply:
Does[ line 40](https://github.com/Financial-Times/dotcom-page-kit/pull/1026/files#diff-07a5a1f5a61d8361b19ebcc04b34f55c76561580f67451783776e7d7e4523d8cR40) need removing?